# Подсказка

- Оберните операцию деления в блок try.
- Перехватите `ArithmeticException` в блоке catch и выведите сообщение об ошибке.
